recipes/usrsctp/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, CMake, tools, AutoToolsBuildEnvironment
+from shutil import copyfile
+import os
+
+
+class ConanRecipe(ConanFile):
+    name = "usrsctp"
+    description = "SCTP user-land implementation (usrsctp)"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sctplab/usrsctp"
+    license = "BSD 3"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "programs": [True, False],
+        "fuzzer": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "programs": False,
+        "fuzzer": False,
+    }
```

Comment:
```suggestion
    }
    exports_sources = "CMakeLists.txt"
```
You have to export your cmake wrapper